### PR TITLE
Add AptosConnect

### PR DESCRIPTION
**Pull request by 0xmaayan**
- Update SDK to version 1.18.1 to include keyless
- Use AptosConnect plugin version 0.0.11

### CODE REVIEW

**Comment by 0xmaayan** [comment]
getting this error when trying to connect AptosConnect on the adapter demo dapp

https://github.com/aptos-labs/aptos-wallet-adapter/assets/29798064/3aba3404-4672-476f-97db-7ae59b20b559

File: packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+  // a newer version (0.1.0) - this causes type mismatch. We should figure out how to handle it.
+  sdkWallets.push(new TWallet() as any);
+} else {
+  sdkWallets.push(new DevTWallet() as any);
+}
+
+let aptosNetwork = isProd ? Network.MAINNET : Network.TESTNET;
+// Need to check window is defined for AptosConnect
+if (typeof window !== "undefined") {
+  sdkWallets.push(new AptosConnectWallet({ network: aptosNetwork }));
```
Comment (hardsetting) [inline packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts:22]: what if the dapp developer wants to use a different config?
e.g. different network, frontendBaseURL or a dappId (to support IC accounts)

Ideally, only if the dapp developer has not registered AptosConnect, then we initialize it as done above.
Might also need to take into consideration race conditions if we have dedupe logic

**Review by hardsetting — APPROVED** [review]

File: apps/nextjs-example/components/transactionFlow/MultiAgent.tsx [hunk 2]
```diff
@@ -23,7 +24,7 @@ export default function MultiAgentTransaction({
     useWallet();
 
   const [secondarySignerAccount, setSecondarySignerAccount] =
-    useState<Account>();
+    useState<Ed25519Account>();
```
Comment (hardsetting) [inline apps/nextjs-example/components/transactionFlow/MultiAgent.tsx:27]: Is this change relevant to the PR?

**Review by alyssaponzo — APPROVED** [review]

File: apps/nextjs-example/components/transactionFlow/MultiAgent.tsx (hunk 2 above)
Comment (0xmaayan) [reply apps/nextjs-example/components/transactionFlow/MultiAgent.tsx:27]: yes, newest sdk version removed the privateKey from Account

File: packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts (hunk 1 above)
Comment (0xmaayan) [reply packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts:22]: got it. we can work on it in a next PR
